src/components/Slices/SlicesTableContainer.jsx

```diff
@@ -0,0 +1,121 @@
+import React from "react";
```

Comment:
This is a new construction not yet anywhere else in Caravan. The idea is for a `container` to be connected to the redux store and so be dependent on the app state. A normal `component` (in this case `SlicesTable.jsx`) just gets passed its state and doesn't care about the app at all. This should make components more reusable.

Reply:
This also has more internal state and manages things like sorting and filtering.